main/src/library/Array.flix

```diff
@@ -408,7 +408,7 @@ mod Array {
     ///
     /// Returns the result of applying `f` to every element in `a`.
     ///
-    /// The result is a new array.
+    /// The result is an empty array.
```

Comment:
@jaschdoc The search and replace here is problematic. In most cases its NOT an empty array. The documentation is trying to indicate that what is returned is new array (but it still contains elements).

Reply:
I will go a head and merge this, but can you restore Array.flix and manually restore the necessary comments in a new PR?

Reply:
> @jaschdoc The search and replace here is problematic. In most cases its NOT an empty array. The documentation is trying to indicate that what is returned is new array (but it still contains elements).

This was deliberate hehe oops